spec/integration_spec.cr

```diff
@@ -195,31 +195,85 @@ describe LuckyRouter do
   end
 
   describe "route with trailing slash" do
-    router = LuckyRouter::Matcher(Symbol).new
-    router.add("get", "/users/:id", :show)
-
     context "is defined with a trailing slash" do
-      router.add("get", "/users/", :index)
-
       it "should treat it as a index route when called without a trailing slash" do
+        router = LuckyRouter::Matcher(Symbol).new
+        router.add("get", "/users/", :index)
```

Comment:
These had to be changed because it was causing the duplicate path error, but was not registering as a failure because it was happening outside of an `it` block